app/assets/javascripts/reducers/article_finder.js

```diff
@@ -140,6 +140,18 @@ export default function articleFinder(state = initialState, action) {
         lastRelevanceIndex: state.lastRelevanceIndex + 50,
       };
     }
+    case RECEIVE_ARTICLE_TITLE: {
```

Comment:
Can this instead be done with the existing `RECEIVE_WIKIDATA_LABELS` action? Using the same data flow as we do with other places that use Wikidata labels would keep things as simple as possible.

Reply:
@ragesoss You mean reusing the same constant to denote the action type in the `article_finder` reducer? Or reusing the `wikidata_labels` reducer (though it doesn't sound possible to do)?

Reply:
I mean re-using the reducer. I think it should be possible, as the strategy is to replace titles with labels (when available) in the individual components. See how they are used in ArticleList and article.jsx, for example.

Reply:
@ragesoss done, reusing the same reducer now. Had to change the `getFilteredArticleFinder` selector so it can mutate the article title from the updated labels